### PR TITLE
TestCase Bounding Box of Line

### PR DESCRIPTION
**Pull request by mehdi138r**
#1-add comment, testcase for bounding box of line shape(cylinder with… 0 radius)

### CODE REVIEW

File: src/tests/bounding_box_test_input.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "cube_1": {
+  "comment":["rotation_angle : the ange of axis(phi and theta) for rotate"
```
Comment (saeedafra) [inline src/tests/bounding_box_test_input.json:3]: typo angle
for rotation. for + a noun. Here "rotate" is a verb form

Comment (saeedafra) [reply src/tests/bounding_box_test_input.json:3]: and anyway it's not clear. angle of what axis?

File: src/tests/bounding_box_test_input.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+            ,"dimension : dimension of shapes that rotate"
```
Comment (saeedafra) [inline src/tests/bounding_box_test_input.json:4]: define each element

File: src/tests/bounding_box_test_input.json [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+            ,"type : type of shape"
+            ,"expected_dimension_box : dimension of bounding box of shape that we expected "],
```
Comment (saeedafra) [inline src/tests/bounding_box_test_input.json:6]: define elements

**Review by saeedafra — CHANGES_REQUESTED** [review]

**Comment by erfanz** [comment]
@mehdi138r 
Using concrete numbers in test cases, especially with multi-digit fractions (e.g. `1.4142135624`), should generally be avoided as it makes the code cryptic and so not checkable by simply reading it. Can you think of a way to avoid it?

**Pull request merged**